.stylelintrc.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
- [ ] Great job setting up linters, Kindly fix all linter errors so that all linters checks will pass. Use `npm hint .` tpo check all webhint errors locally and `npx stylelint "**/*.{css,scss}"` to check all stylelint errors locally.